oot.py

```diff
@@ -0,0 +1,104 @@
+import copy
+from typing import NamedTuple
+
+import bpy
+import mathutils
+
+from .material.parser import parse_f3d_rendermode_preset, F64RenderState
+from .common import ObjRenderInfo, draw_f64_obj, collect_obj_info, get_scene_render_state
+from .properties import F64RenderSettings
+from .globals import F64_GLOBALS
+
+class RoomRenderInfo(NamedTuple):
+  render_state: F64RenderState
+  name: str
+  def __hash__(self):
+    return hash(self.name)
+
+def get_oot_room_childrens(scene: bpy.types.Scene):
+  if F64_GLOBALS.oot_room_lookup is not None:
+    return F64_GLOBALS.oot_room_lookup
+
+  oot_room_lookup = {}
+  render_state = get_scene_render_state(scene)
+  scene_objs: list[bpy.types.Object] = []
+  room_objs: list[bpy.types.Object] = []
+
+  def get_room_children(obj: bpy.types.Object, name: str = ""):
```

Comment:
```suggestion
  def get_room_children(obj: bpy.types.Object, name: str):
```
having the default is confusing (afaict it's unused)